recipe/0125-rendering-3d/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Including a 3D Rendering
+id: 93
+layout: recipe
+tags: [tbc]
+summary: "You want to include a 3D model representation"
+---
+
+
+## Use Case
+
+You want to view an object from any orbital point of view or zoom level.
+You want to view an object in augmented reality/XR in order to gain a better understanding of its dimensions and physical properties.
+
+## Implementation notes
+
+Add a [`rendering`](https://iiif.io/api/presentation/3.0/#rendering) to your manifest either at the root level, or any applicable child resource.

```

Comment:
Could you go into more detail about the rendering property. Also when would it make sense to have it at the root level and when at a canvas?